src/staging/validation.py

```diff
@@ -156,7 +156,8 @@ def validate_data_with_schema(survey_df: pd.DataFrame, schema_path: str):  # noq
                         " the data."
                     )
             else:
-                survey_df[column] = survey_df[column].astype(dtypes_dict[column])
+                if survey_df[column].isna().all() is False:
+                    survey_df[column] = survey_df[column].astype(dtypes_dict[column])
```

Comment:
The reason for this change is that unlike BERD backdata, PNP  backdata contains columns with only nan values. When the original line 159 would run it would throw an error because of this. Line 159 now checks if the column does not only contain nan values, and if so executes line 160 as normal.